Warehouse.Persistence.EF/Persistence/Contexts/WarehouseDbContext.cs

```diff
@@ -0,0 +1,175 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Warehouse.Domain.Entities;
+using Warehouse.Persistence.EF.Configuration.Entities;
+
+namespace Warehouse.Persistence.EF.Persistence.Contexts;
+
+public class WarehouseDbContext : DbContext
+{
+    public WarehouseDbContext(DbContextOptions options) : base(options)
+    {
+    }
+
+    public DbSet<User> Users { get; set; }
+    public DbSet<Order> Orders { get; set; }
+    public DbSet<OrderLine> OrderLines { get; set; }
+    public DbSet<OrderStatus> OrderStatus { get; set; }
+    public DbSet<Payment> Payments { get; set; }
+    public DbSet<Group> Groups { get; set; }
+    public DbSet<Size> Sizes { get; set; }
+    public DbSet<ProductSize> ProductSizes { get; set; }
+    public DbSet<Product> Products { get; set; }
+    public DbSet<ProductGroup> ProductGroups { get; set; }
+    public DbSet<Brand> Brands { get; set; }
+    public DbSet<UserRole> UserRoles { get; set; }
+    public DbSet<Basket> Baskets { get; set; }
+    public DbSet<BasketLine> BasketLines { get; set; }
+    public DbSet<Role> Roles { get; set; }
+
+    protected override void OnModelCreating(ModelBuilder builder)
+    {
+        builder.ApplyConfiguration(new UserEntityConfiguration());
+        builder.ApplyConfiguration(new ProductEntityConfiguration());
+        builder.ApplyConfiguration(new ProductSizeEntityConfiguration());
+        builder.ApplyConfiguration(new ProductGroupEntityConfiguration());
+        builder.ApplyConfiguration(new OrderEntityConfiguration());
+        builder.ApplyConfiguration(new OrderLinesEntityConfiguration());
+        builder.ApplyConfiguration(new UserRoleEntityConfiguration());
+        builder.ApplyConfiguration(new BasketLineEntityConfiguration());
+
+        // Seed Brands
+        builder.Entity<Brand>().HasData(
```

Comment:
Let the seed be in some separate class that handles the seeding of the data. Or a service that later if you need to seed a huge amount of data